@navikt/core/css/darkside/button.darkside.css

```diff
@@ -33,99 +33,95 @@
+  border-radius: var(--ax-border-radius-medium);
   display: inline-flex;
   cursor: pointer;
-  margin: 0;
   text-decoration: none;
   border: none;
   background: none;
-  border-radius: var(--ac-button-border-radius, var(--a-border-radius-medium));
-  padding: var(--ac-button-padding, var(--__ac-button-padding));
   align-items: center;
   justify-content: center;
-  gap: var(--a-spacing-2);
+  gap: var(--ax-spacing-2);
+  font-weight: 600;
+
+  /* TODO: fonts should be done globally elsewhere */
+  font-size: 18px;
+  line-height: 24px;
+  font-family: "Source Sans 3", "Source Sans Pro", Arial, sans-serif;
 }
 
 .navds-button--small {
-  --__ac-button-padding: var(--a-spacing-1) var(--a-spacing-3);
-
-  padding: var(--ac-button-padding-small, var(--__ac-button-padding));
+  padding: var(--ax-spacing-1) var(--ax-spacing-3);
   min-height: 2rem;
   min-width: 2rem;
 }
 
 .navds-button--xsmall {
-  --__ac-button-padding: var(--a-spacing-05) var(--a-spacing-2);
-
-  padding: var(--ac-button-padding-xsmall, var(--__ac-button-padding));
-  gap: var(--a-spacing-1);
+  padding: var(--ax-spacing-05) var(--ax-spacing-2);
+  gap: var(--ax-spacing-1);
 }
 
 .navds-button--icon-only {
-  --__ac-button-padding: var(--a-spacing-3);
-
-  padding: var(--ac-button-padding-icon-only, var(--__ac-button-padding));
+  padding: var(--ax-spacing-3);
 }
 
 .navds-button--small.navds-button--icon-only {
-  --__ac-button-padding: var(--a-spacing-1);
-
-  padding: var(--ac-button-padding-icon-only-small, var(--__ac-button-padding));
+  padding: var(--ax-spacing-1);
 }
 
 .navds-button--xsmall.navds-button--icon-only {
-  --__ac-button-padding: var(--a-spacing-05);
-
-  padding: var(--ac-button-padding-icon-only-xsmall, var(--__ac-button-padding));
+  padding: var(--ax-spacing-05);
 }
 
 .navds-button:focus-visible {
-  outline: 2px solid transparent;
+  outline: 4px solid var(--ax-border-focus);
   outline-offset: 2px;
-  box-shadow: var(--a-shadow-focus);
 }
 
 @supports not selector(:focus-visible) {
   .navds-button:focus {
-    outline: none;
-    box-shadow: var(--a-shadow-focus);
+    outline: 4px solid var(--ax-border-focus);
   }
 }
 
 .navds-button__icon {
-  --ac-button-icon-margin: -4px;
-
   font-size: 1.5rem;
   display: flex;
 }
 
+:root {
+  --__axc-button-icon-margin: -4px;
+}
+
 .navds-button__icon:first-child {
-  margin-left: var(--ac-button-icon-margin);
+  margin-left: var(--__axc-button-icon-margin);
 }
 
 .navds-button__icon:last-child {
-  margin-right: var(--ac-button-icon-margin);
+  margin-right: var(--__axc-button-icon-margin);
 }
 
-.navds-button__icon:only-child {
-  margin: 0;
+:where(.navds-button--xsmall, .navds-button--small) .navds-button__icon:first-child,
+:where(.navds-button--xsmall, .navds-button--small) .navds-button__icon:last-child {
+  --__axc-button-icon-margin: -2px;
 }
 
 :where(.navds-button--xsmall, .navds-button--small) .navds-button__icon {
-  --ac-button-icon-margin: -2px;
-
   font-size: 1.25rem;
 }
```

Comment:
These could be merged (don't need to specify first-child and last-child for the variable)